list_people.rb

```diff
@@ -0,0 +1,9 @@
+module ListPeople
+  def list_all_people
+    users.each do |user|
+      puts "[#{user.classroom}] Name: #{user.name} | ID: #{user.id} | Age: #{user.age}"
+    end
+    puts "\n"
+    continue?
+  end
+end
```

Comment:
Similarly, option 5 for listing all people is not working as well. Kindly look into it.

![image](https://user-images.githubusercontent.com/38088818/174107219-98c7a4f2-1b71-4ecb-bf24-dbf0c5aa7405.png)

I couldn't check if option 6 for listing all rentals by ID because the option for creating people and listing all people is not working therefore I cannot access the IDs.

- When you have resolved the above mentioned points, kindly make sure that option 6 is working properly as well.